src/main/java/com/github/manolo8/darkbot/utils/login/LoginUtils.java

```diff
@@ -158,10 +166,16 @@ private static void usernameLogin(LoginData loginData, String domain) throws IOE
 
         CookieHandler.setDefault(null);
 
-        HttpCookie cookie = cookieManager.getCookieStore().getCookies().stream()
+        HttpCookie cookie;
+
+        if (captcha && domain.equals("lp")) {
+            Application.launch(LoginBrowser.class, loginData.getUsername(), loginData.getPassword());
+        }
```

Comment:
You can know there's a captcha before trying to log in, why do we try to log in (knowing it will fail) before doing this?

You have the CaptchAPI a bit higher up, which is the *perfect* place to just add this handling code